cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -128,6 +128,8 @@
     public static final String MESSAGING_SERVICE = "messaging.service";
     public static final String RUNTIME = "runtime";
     public static final String AUTHENTICATION = "authentication";
+    public static final String TASK_DISPATCHER = "task.dispatcher";
```

Comment:
Unused?

Reply:
Yes. We removed the dispatcher part. Thx for catching this.